linux_os/guide/system/software/disk_partitioning/encrypt_partitions/oval/shared.xml

```diff
@@ -0,0 +1,50 @@
+<def-group>
```

Comment:
I suggest adding a warning to the rule.yml that the rule doesn't have a remediation.